docs: Add 2025-10-16 meeting notes (#625)
eslint/tsc-meetings: notes/2025/2025-10-16.md

# 2025-10-16 ESLint TSC Meeting Notes

## Transcript

> [`2025-10-16-transcript.md`](../../transcript/2025-10-16-raw.md)

Reviewer: ```suggestion
[`2025-10-16-transcript.md`](2025-10-16-transcript.md)
```

Copy-paste error?

**TSC Question:** What should be done with these inactive RFCs?

**Resolution:** The TSC agreed on the following approach for both RFCs:
- **RFC #114 (Relative cache support):** Give the author one week to see if they'd like to pick it up. If there's no response after a week, @mdjermanovic will open a new RFC to continue the work.
- **RFC #128 (Localization):** Post on the RFC to see if the author still wants to continue with it. If after a week there's no response, the RFC will be closed.

### V10 Development Planning

The TSC reviewed the [v10.0.0 project board](https://github.com/orgs/eslint/projects/6/views/1) and discussed several items.

#### [Rule Change: `radix` rule should have better options](https://github.com/eslint/eslint/issues/19916)

> **TSC Summary:** This issue was missed in the last meeting. It proposes implementing suggestions for the `radix` rule with a new option.

Reviewer: ```suggestion
**TSC Summary:** This issue was missed in the last meeting. It proposes modifying options for the `radix` rule.
```
